Pipfile

```diff
@@ -0,0 +1,39 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+"psycopg2" = "*"
+"psycopg2-binary" = "*"
+gunicorn = "*"
+djangorestframework = "*"
+celery = "*"
+"7b96e2b" = {file = "file:///Users/cadams/Projects/concordia/vendor/django-registration.zip"}
```

Comment:
Can this not be an absolute path? How would this translate to another environment?